presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/PrestoSparkTaskInputs.java

```diff
@@ -20,6 +20,7 @@
 import java.util.HashMap;
```

Comment:
Appologies that I don't have the knowledge to review the code in classloader_interface. From a glance it looks suspecious to have this intrusive changes (having read/write descriptor propagating this far). But I'm likely wrong. Can we find another person to review this part of the code? Thanks!